applications/ehr_query_llm/fhir/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.20)
+project(fhir LANGUAGES NONE)
+
+# Install application and dependencies into the install/ directory for packaging
+install(
+  FILES __main__.py
+        app.yaml
+        fhir_client.py
+        metadata.json
+        packageHAP.sh
+        README.md
+        requirements.txt
+        run_cmd_template.sh
+        test_fhir_client.py
+  DESTINATION bin/fhir/python
+  COMPONENT "holohub-apps"
+)
+
+install(
+  DIRECTORY ${CMAKE_HOME_DIRECTORY}/operators/ehr_query_llm
+  DESTINATION bin/fhir/python/operators)
```

Comment:
Is it missing a `) here, or is the file incomplete?

Reply:
Formatting fixup wouldn't hurt but not a blocker.

A couple of requests:
1. Could you please update `CMAKE_HOME_DIRECTORY` to `CMAKE_SOURCE_DIR` or omit it entirely? `CMAKE_HOME_DIRECTORY` is an internal detail ([doc](https://cmake.org/cmake/help/latest/variable/CMAKE_HOME_DIRECTORY.html))
2. Please move the `install` directive to `operators/ehr_query_llm/CMakeLists.txt` for best practice so that the operator folder is responsible for its own installation in case of reuse in other apps

Reply:
Sorry, for some reason, I didn't see the closing ')' in the review mode :see_no_evil: